frontend/src/components/Forms/TeamEventCreditsForm/TeamEventCreditsForm.tsx

```diff
@@ -202,6 +207,7 @@ const TeamEventCreditForm: React.FC = () => {
           approvedAttendance={approvedAttendance}
           pendingAttendance={pendingAttendance}
           isAttendanceLoading={isAttendanceLoading}
+          // teamEventAttendance={teamEventAttendance}
```

Comment:
See my comment below, but if you're passing in `setApprovedAttendance in this prop`, you'd do it here

i.e.
```
pendingAttendance={pendingAttendance}
isAttendanceLoading={isAttendanceLoading}
// add the following line
setPendingAttendance={setPendingAttendance}
```